fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1250,4 +1250,16 @@
      */
     @ConfField(mutable = true, masterOnly = true)
     public static boolean enable_batch_delete_by_default = false;
+
+    /**
+     * Used to set default db data quota bytes.
+     */
+    @ConfField(mutable = true, masterOnly = false)
+    public static long default_db_data_quota_bytes = 1024 * 1024 * 1024 * 1024L; // 1TB
+
+    /**
+     * Used to set default db replica quota bytes.
+     */
+    @ConfField(mutable = true, masterOnly = false)
+    public static long default_db_replica_quota_bytes = 1024 * 1024 * 1024;
```

Comment:
The unit is wrong. This is not a "byte size", but a "number of" replicas.

Reply:
I restore the modification of default_db_replica_quota_size.